lib/models/Mailbox.js

```diff
@@ -83,6 +83,30 @@ export default class Mailbox extends Model {
   }
 
   /**
+   * Update this {@link Mailbox}.
+   *
+   * @param options {Object} The attribute to be updated in this {@link Mailbox}.
```

Comment:
Please document that we can only pass `options.name` and `options.parentId`. See jsDoc documentation for details, or other examples in the code.